docs/gitops/README.md

```diff
@@ -10,6 +10,40 @@ oc apply -f docs/gitops/gitops-subscription.yaml
 # ...
 oc get csv -n openshift-gitops -w
 ```
+### Install Serverless Catalog Source
+
+If you are developing the Serverless operator, you can use the local catalog source as follows.
+
+```shell
+# Install the local catalog index
+export DOCKER_REPO_OVERRIDE=...
+make image; make install-with-argo-cd
+
+# Modify the 100-subscriptions.yaml to use the local catalog source
+# add the CatalogSource
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
```

Comment:
The image is used when we define the DOCKER_REPO_OVERRIDE var. The exact value  is not relevant.
In my case I just had it pointing to my docker repo. See [here](https://github.com/openshift-knative/serverless-operator/blob/main/hack/lib/catalogsource.bash#L29), no?

Reply:
> The exact env var value is not relevant

You're right. My bad.

The command `make dev` will install the (wrong) CatalogSource but we suggest modifying so it will overwrite the wrong catalogsource later and the correct one will be deployed. I guess that's ok.